gulpfile.js

```diff
@@ -441,6 +441,7 @@ function copyCss() {
  * @return {!Promise}
  */
 function watch() {
+  printConfigHelp('gulp watch', 'dist/amp.js');
```

Comment:
Consider moving `dist/amp.js` to a shared const.

Reply:
Done.